CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 #### Features
 
 * [#1686](https://github.com/ruby-grape/grape/pull/1686): Avoid coercion of a value if it is valid - [@timothysu](https://github.com/timothysu).
+* [#1688](https://github.com/ruby-grape/grape/pull/1688): Fixes #1090 - [@ramkumar-kr](https://github.com/ramkumar-kr).
```

Comment:
Lets be explicit here, make it "Removed Yard docs".

Reply:
done